one_fm/hooks.py

```diff
@@ -218,9 +218,6 @@
 		"before_submit": "one_fm.purchase.utils.before_submit_purchase_receipt",
 		"on_submit": "one_fm.one_fm.doctype.customer_asset.customer_asset.on_purchase_receipt_submit"
 	},
-	"ToDo": {
-		"after_insert": "one_fm.grd.utils.todo_after_insert"
-	},
```

Comment:
Need this to be preserved

Reply:
@pjamsheer no need to be stored in todo list.